### PR TITLE
PV Database Update

### LINKED ISSUE #521: update Hive PV catalogue
Update the PV technologies catalogue in the Energy Systems Hizard:

image

That includes:
- [x] reviewing existing module types and if necessary removing / adding tech
- [x] remove HIT PV from catalogue, because it is actually proprietory (Panasonic?) (https://miro.com/app/board/o9J_leSAuA0=/)
- [x] update parameters (module efficiency, spec embod emissions, spec capital cost, tech description

Database to be worked on: 
- `.../src/Hive.IO/Hive.IO/Resources/pv_efficiency.csv` (tech parameters)
- `.../src/Hive.IO/Hive.IO/Resources/EnergySystems` (images)
- `.../src/Hive.IO/Hive.IO/EnergySystems/surface_tech_module_types.json` (contains the current descriptions. Could you add a `Description` column to the `pv_efficiency.csv` above and add a string there? Note: no need to edit this `.json`, because there should be a script that creates it based on the `csv`) 
- `.../src/Hive.IO/Hive.IO/Resources/Sources.docx` (document your references used in this file)

### PR DESCRIPTION
**Pull request by justinfmccarty**
# Update PV Database

## Issues

Closes #521 

## Description

1. Removed existing PV technologies
2. Added new PV technologies. The new file has the same field names as the old but I have included much more information in it including sources and methodology for calculating each technologies GHGs, Costs, efficiency, etc.
3. Removed old pictures and added new pictures
4. Added photoshop (.psd) files in directory containing PV pictures to .gitignore

## Checklist

- [x] Remove debugging artifacts (if needed)
- [ ] Rebuild the Setup_Hive.exe
- [ ] Update Hive Wiki (if needed)
- [ ] Update most important Grasshopper template(s); as of now [/GrasshopperExamples/LectureExercises/EaCS3_E04_Hive_Template.gh](https://github.com/architecture-building-systems/hive/blob/master/GrasshopperExamples/LectureExercises/EaCS3_E04_Hive_Template.gh)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -141,3 +141,4 @@ Ankh.NoLoad
 
 src/Hive.IO/packages/
 .vscode/
+src/Hive.IO/Hive.IO/Resources/EnergySystems/*.psd
```
Comment (christophwaibel) [inline .gitignore:144]: why put psd files into gitignore? Let's keep them in the repo

Comment (christophwaibel) [reply .gitignore:144]: are the images used open access? We should be careful with copyright

File: src/Hive.IO/Hive.IO/Resources/pv_efficiency.csv [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+roof-intergrated Monocrystalline,0.171428571,0,657.3705179,3834.661355,52.30769231,425,40,This is a standard roof panel system based on a monocrystalline cell construction. It can be easily applied in new construction and retrofit situations.,roof-intergrated,Monocrystalline Roof Panel,Monocrystalline,None,22.8,1300x875x9 mm,1.1375,20.04395604,195,17.14285714,3.834661355,657.3705179,171.4285714,52.30769231,425,Megaslate Fassade,https://3s-solarplus.ch/wp-content/uploads/2021/07/DB_F_MS-Fassade_DE_2020-11.pdf,Switzerland,Cost calculated based on coefficient dervied from ratio of a-Si integrated and a-Si attached (proxy for labor) applied to base monocrystalline cost,Derived from STC on Power Rating,Same as white monocrystalline,mono_roof
```
Comment (christophwaibel) [inline src/Hive.IO/Hive.IO/Resources/pv_efficiency.csv:27]: there is a problem with unicode characters: � 
maybe some letter from mac?

Comment (christophwaibel) [reply src/Hive.IO/Hive.IO/Resources/pv_efficiency.csv:27]: wow this list is long! @justinfmccarty could you please split this csv into "pv_efficiency.csv" and "bipv_efficiency.csv"? Let us make BIPV its own category in Hive :-)

**Review by christophwaibel — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (justinfmccarty) [reply .gitignore:144]: I will redo the images.

**Pull request merged**